src/expr.rs

```diff
@@ -78,12 +78,10 @@ pub(crate) fn format_expr(
         ast::ExprKind::Lit(ref l) => {
             if let Some(expr_rw) = rewrite_literal(context, l, shape) {
                 Some(expr_rw)
+            } else if let LitKind::StrRaw(_) = l.token.kind {
```

Comment:
warning: this `else { if .. }` block can be collapsed
  --> src\expr.rs:81:20
```
   |
81 |               } else {
   |  ____________________^
82 | |                 if let LitKind::StrRaw(_) = l.token.kind {
83 | |                     Some(context.snippet(l.span).trim().into())
84 | |                 } else {
85 | |                     None
86 | |                 }
87 | |             }
   | |_____________^
   |
```
   = note: `#[warn(clippy::collapsible_else_if)]` on by default
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#collapsible_else_if